packages/flutter/test/material/text_field_test.dart

```diff
@@ -1985,7 +1949,7 @@ void main() {
 
     // Selection should stay the same since it is set on tap up for mobile platforms.
     await touchGesture.down(gPos);
-    await tester.pumpAndSettle();
+    await tester.pump();
```

Comment:
Had to change 2 tests since the test gesture was meant to be a tap but ended up being a long press.

Reply:
@Renzo-Olivares are you ok with this change?